tesseract/tesseract_planning/tesseract_motion_planners/include/tesseract_motion_planners/trajopt/config/trajopt_planner_config.h

```diff
@@ -49,6 +49,9 @@ struct TrajOptPlannerConfig
    */
   virtual bool generate();
 
+  /** @brief The QP solver used in the SQP optimization routine */
+  sco::ModelType optimizer = sco::ModelType::AUTO_SOLVER;
```

Comment:
So I thought about adding this here, but I decided against it because model is usually set when you construct the problem. So a planner that only has the trajopt_planner_config would have to decide whether to use the model already in the problem or the model selected here.

Reply:
I see your point. Should we then move params and callbacks out of this class and into the default for the same reason?